syntax = "proto3";

package sn;

message SysCmd 
{
	string id = 1; //随机字符串
	bytes forkid = 2;
	int32 cmd = 3;
	repeated string arg = 4;
}

message TxCmd
{
	string id = 1; //随机字符串
	bytes forkid = 2; //分支id
	bytes hash = 3; //tx hashid
}

message BlockCmd
{
	string id = 1;  //随机字符串
	bytes forkid = 2; //分支id
	bytes hash = 3; //区块hash id
}

message VPeerNetEvent
{
    bytes data = 1; // 虚拟PeerNet二进制数据
}
